index.html

```diff
@@ -5,14 +5,135 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
     <!-- Add a link to your custom CSS here -->
-
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
-<body>
-
+<body class="container">
+    <header class="row">
+        <div class="col-3">
+            <p class="name">Alice Rose</p>
+            <p class="job">Personal Life Coach</p>
+        </div>
+        <nav class="col-6">
+            <ul class="navlist">
+                <li>Home</li>
+                <li>About</li>
+                <li>Services</li>
+                <li>Motivational Reading</li>
+                <li>Blog</li>
+                <li>Contact</li>
+            </ul> 
+        </nav>
+        <div class="row col-3">
+            <i class="col-4 fa fa-user-circle-o"></i>
+            <p class="col-8 login">Log in</p>
+        </div>
+    </header>
+    <section class="intro">
+        <div>
+            <p class="ambition">AMBITION IS THE FIRST STEP TOWARDS</p>
```

Comment:
I think it makes more sense to use an `id` here because there is only one of the type of text

Reply:
I have replaced it with id.